hrcase/CRM/HRCase/Upgrader.php

```diff
@@ -367,25 +376,27 @@ private function createOrUpdateDefaultCaseTypes($defaultCaseTypes) {
    * with this extension if they are not exits
    *
    * @param array $defaultActivityTypes
-   *   A list of activity type names
+   *   A list of activity types grouped by component name
    */
-  private function up1402_createDefaultActivityTypes($defaultActivityTypes) {
-    foreach ($defaultActivityTypes as $activityType) {
-      $type = civicrm_api3('OptionValue', 'get', [
-        'sequential' => 1,
-        'name' => $activityType,
-        'option_group_id' => 'activity_type',
-        'component_id' => 'CiviTask',
-      ]);
-
-      if (empty($type['id'])) {
-        civicrm_api3('OptionValue', 'create', [
+  private function createOrUpdateActivityTypes($defaultActivityTypes) {
```

Comment:
there is no update happening here just create .. so it should be **createActivityTypes**